Bottom: a6d47921e703dae4d292502c2ae5dc189af379e0
Top:    d8b2e1f710e6e68b23e47bbd3759a1b0e1d5de69
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:08:38 +0000

Refresh of extend-object3d_from_contours

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a593a56..8b0f71b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -396,7 +396,6 @@ object3d_from_contours (const POLYAREA *contours,
       edges    = malloc (sizeof (edge_ref  ) * 3 * npoints); /* (n-bottom, n-top, n-sides) */
       faces    = malloc (sizeof (face3d *) * (npoints + 2)); /* (n-sides, 1-bottom, 1-top */
 
-<<<<<<< current
       /* Define the vertices */
       ct = contour;
       offset_in_ct = 0;
